go/base/static/js/src/components/views.js

```diff
@@ -42,10 +42,30 @@
           $p = $('<p class="textarea-char-count"/>');
           this.$el.after($p);
       }
-      this.totalChars = this.$el.val().length;
-      this.totalSMS = Math.ceil(this.totalChars / this.SMS_MAX_CHARS);
-      $p.html(this.totalChars + ' characters used<br>' + this.totalSMS + ' smses');
-
+      var text = this.$el.val();
+      var non_ascii = go.utils.non_ascii(text);
+      this.containsNonAscii = (non_ascii.length > 0);
+      this.totalChars = text.length;
+      this.totalBytes = text.length * (1 + this.containsNonAscii);
+      this.totalSMS = Math.ceil(this.totalBytes / this.SMS_MAX_CHARS);
+      var html;
+      if (this.containsNonAscii) {
+          html = [
+              'Non-ASCII characters: ' + non_ascii.join(', '),
+              this.totalChars + ' characters used (~' +
+                  this.totalBytes + ' bytes)',
```

Comment:
Happy for this to be bike shedded if people have concrete suggestions.

I'm personally okay with using the word bytes because:

* It's only there if you have non-ASCII characters.
* The SMS count is the most important part.
* If you're sending SMSes with non-ASCII characters you need to have some
idea of what bytes are anyway, because they directly affect how much money
you're going to spend (and spending $200,000 instead of $100,000 is enough
reason to learn the meaning of "bytes").